### PR TITLE
19 weather api to html

### PR DESCRIPTION
**Pull request by Epanastatis2020**
Matched up the weather API data with the HTML

Changed the styling and added functionality to empty the weather container on each new search.

Also had to make some amendments to the HTML including removing some divs and allocating IDs where necessary

Finally, I added a function around the newsAPI ajax call and passed it the country code from the weatherAPI.

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -1,87 +1,101 @@
 // NewsAPI
 
-let strNewsAPIKey = "0e966fb836610aa7a1a213a0b9d61c6b";
-let strCountryCode = "au";
+let newsAPIKey = "0e966fb836610aa7a1a213a0b9d61c6b";
+let countryCode = "au";
 let newsQueryURL = `https://gnews.io/api/v4/top-headlines?country=${countryCode}&max=5&token=${newsAPIKey}`;
```
Comment (amightyrose) [inline assets/js/script.js]: - The newsAPIKey and newsQueryURL variables need to be moved into the getNews function which I forgot to do earlier.
- The ${countryCode} variable inside the newsQueryURL string needs to be changed to 'catchCountry' to match the variable name you're passing from your function.
- The countryCode variable declaration can be deleted entirely.

**Review by amightyrose — CHANGES_REQUESTED** [review]
Have put a couple of notes on things that need to be updated. Also the console.log lines that I accidentally left in can be removed. They're not needed any more. Thanks.

File: assets/js/script.js (hunk 1 above)
Comment (Epanastatis2020) [reply assets/js/script.js]: Have made the appropriate changes, please review and comment/approve as necessary.

**Review by amightyrose — APPROVED** [review]
Nice work!

**Pull request merged**